setup.cfg

```diff
@@ -1,46 +1,2 @@
 [flake8]
```

Comment:
If I'm installing an extra package to use coverage in toml, would actually make sense to do the same for flake8

Reply:
done, setup.cfg completely remvoed